src/interpreter/naive_interpreter.nit

```diff
@@ -903,6 +978,16 @@ redef class AMethPropdef
 				return v.float_instance(args[0].to_i.to_f)
 			else if pname == "to_b" then
 				return v.byte_instance(args[0].to_i.to_b)
+			else if pname == "to_i8" then
```

Comment:
Unfortunately, no they cannot be :/

Java does not support unsigned fix integers, we could work around it, but we should first discuss how it should be done